html/changelogs/AuroraBirdie-PR-#16055.yml

```
@@ -0,0 +1,63 @@
+################################
+# Example Changelog File
+#
+# Note: This file, and files beginning with ".", and files that don't end in ".yml" will not be read. If you change this file, you will look really dumb.
+#
+# Your changelog will be merged with a master changelog. (New stuff added only, and only on the date entry for the day it was merged.)
+# When it is, any changes listed below will disappear.
+#
+# Valid Prefixes:
+#   bugfix
+#   wip (For works in progress)
+#   tweak
+#   soundadd
+#   sounddel
+#   rscadd (general adding of nice things)
+#   rscdel (general deleting of nice things)
+#   imageadd
+#   imagedel
+#   maptweak
+#   spellcheck (typo fixes)
+#   experiment
+#   balance
+#   admin
+#   backend
+#   security
+#   refactor
+#################################
+
+# Your name.
+author: AuroraBirdie
+
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - maptweak: "Changed Command chairs with pilot chairs" 
+  - maptweak: "Moved Sensor and Thruster Console back to original location"
+  - maptweak: "removed plastic chairs by helm"
+  - maptweak: "removed central pillar"
+  - maptweak: "removed cameras and lights from central pillar"
+  - maptweak: "added cameras above key auths"
+  - maptweak: "moved ship weapons, levi key, and levi button into old sec cam location"
+  - maptweak: "moved sec cam, station alert, and med console to where I wanted to put thruster/sensors (left side of middle chair)"
+  - maptweak: "added a 'Captain's Workstation' where old pillar was"
+  - maptweak: "added extra lockdown and control buttons"
+  - maptweak: "removed downward floor stairs"
+  - maptweak: "removed downward floor stair left of helm down"
+  - maptweak: "added railing around new extension"
+  - maptweak: "removed intercomms to old bridge crew stations. Put intercomm by sensor console"
+  - maptweak: "added extra chair for seating choices and slight requirements"
+  - maptweak: "added pillar behind captain's chair for a camera and lights"
+  - maptweak: "added extra vent"
+  - maptweak: "Hid all vents and scrubbers"
+  - maptweak: "added plants by consoles"
+  - maptweak: "removed one of the paperbins/folders/pen sets"
+  - maptweak: "messaging/engineer console put in back left"
+  - maptweak: "removed all consoles from top right, as all other stations were handled. Put decorations there instead"
+  - maptweak: "added naviation console to the Captain's seat- made it more symmerical"
```

```suggestion
  - maptweak: "Remapped the bridge."
```
no. please shorten this changelog to a single entry.